development/client/src/App.js

```diff
@@ -0,0 +1,51 @@
+import React, { useRef } from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link
+} from "react-router-dom";
+import { useQuery, useApolloClient } from '@apollo/client';
+import ColorSchemeGenerator from './ColorSchemeGenerator';
+import Favorites from './Favorites';
+import ColorLookup from './ColorLookup';
+import { GET_SAVED_COLORS } from './queries';
+import './App.css';
+
+function App() {
+  const inputRef = useRef(null);
+  useQuery(GET_SAVED_COLORS);
+  const client = useApolloClient();
```

Comment:
It looks like this line (and the `useApolloClient` import) can be removed (`client` isn't being used).